src/ucp/core/ucp_ep.c

```diff
@@ -1634,25 +1647,23 @@ static void ucp_ep_config_print(FILE *stream, ucp_worker_h worker,
     fprintf(stream, "#\n");
 
     if (context->config.features & UCP_FEATURE_TAG) {
-         tag_config = (ucp_ep_is_tag_offload_enabled((ucp_ep_config_t *)config)) ?
-                       &config->tag.eager : &config->am;
-         ucp_ep_config_print_tag_proto(stream, "tag_send",
-                                       tag_config->max_short,
-                                       tag_config->zcopy_thresh[0],
-                                       config->tag.rndv.rma_thresh,
-                                       config->tag.rndv.am_thresh);
-         ucp_ep_config_print_tag_proto(stream, "tag_send_nbr",
-                                       tag_config->max_short,
-                                       /* disable zcopy */
-                                       config->tag.rndv_send_nbr.rma_thresh,
-                                       config->tag.rndv_send_nbr.rma_thresh,
-                                       config->tag.rndv_send_nbr.am_thresh);
-         ucp_ep_config_print_tag_proto(stream, "tag_send_sync",
-                                       tag_config->max_short,
-                                       tag_config->sync_zcopy_thresh[0],
-                                       config->tag.rndv.rma_thresh,
-                                       config->tag.rndv.am_thresh);
-     }
+        ucp_ep_config_print_tag_proto(stream, "tag_send",
+                                      config->tag.eager.max_short,
```

Comment:
is this a bugfix that we printed am config instead of tag config?

Reply:
yes, config->am_config has not adjusted limits
`config->tag.eager` has to be used here

Reply:
it was an equivalent before this change. BTW, now when HWTM is off you adjust `tag.eager.config`, but does not adjust `am.config` (tag.eager is set to am before adjustments). Can it be a problem? Do we use `am` config thresholds in the code somewhere?

Reply:
yes, looks like we are using "tag"

Reply:
ok